File: packages/lib-shared/src/core/utils/pick/pick.models.ts
import { type _PickModel, type _PickParamsModel } from '@lib/shared/core/utils/pick/_pick.models';

export type PickParamsModel<TType extends object, TKey extends string> = _PickParamsModel<
  TType,
  TKey
>;

export type PickModel<TType extends object, TKey extends string> = _PickModel<TType, TKey>;

File: packages/lib-shared-py/src/lib_shared/core/utils/pick/pick_models.py
from typing import Any, Mapping

from lib_shared.core.utils.pick._pick_models import _PickModel, _PickParamsModel

type PickParamsModel[T: Mapping[str, Any]] = _PickParamsModel[T]

type PickModel[T: Mapping[str, Any]] = _PickModel[T]
